### ISSUE TITLE
Installer no longer installing on linux

### ISSUE THREAD

**Issue by b3arpsl** [zebrium/ze-fluentd-plugin#2]
When trying to install the collector on a linux machine, we get the following error message: 

```
Installer versoin 1.49.3
OS information:
uname -a
Linux zebriumsyslog 5.15.0-25-generic #25-Ubuntu SMP Wed Mar 30 15:54:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
cat /etc/os-release
PRETTY_NAME=“Ubuntu 22.04 LTS”
NAME=“Ubuntu”
VERSION_ID=“22.04”
VERSION=“22.04 (Jammy Jellyfish)”
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL=“https://www.ubuntu.com/”
SUPPORT_URL=“https://help.ubuntu.com/”
BUG_REPORT_URL=“https://bugs.launchpad.net/ubuntu/”
PRIVACY_POLICY_URL=“https://www.ubuntu.com/legal/terms-and-policies/privacy-policy”
UBUNTU_CODENAME=jammy
cat /etc/issue
Ubuntu 22.04 LTS \n \l
Installing package dependies
Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
Get:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [109 kB]
Get:4 http://archive.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
Fetched 338 kB in 0s (703 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
build-essential is already the newest version (12.9ubuntu3).
ruby-dev is already the newest version (1:3.0~exp1).
0 upgraded, 0 newly installed, 0 to remove and 104 not upgraded.
Installing log collector dependencies
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    41  100    41    0     0    313      0 --:--:-- --:--:-- --:--:--   315
sh: 1: GET: not found
```

**Comment by SeamusClark** [comment]
Tracking

**Comment by b3arpsl** [comment]
Looks like this is using the EOL'd td-agent-3 file. Because this install is using Jammy, when it constructs the url path for download from treasure data, it is looking up https://toolbelt.treasuredata.com/sh/install-ubuntu-jammy-td-agent3.sh, which does not exist.  This should be updated to td-agent-4 anywho since 3 is EOL'd

**Comment by b3arpsl** [comment]
So a bunch of stuff ended up getting wrapped up in this one issue, as it was a can of worms once I cracked it open to swap out the agents.  Here is a short list of what was changed:

* Onboarded support for Oracle Linux
* Added initial test automation to test that the installer will install on every supported OS 
* Improved logic in script to catch edgecases and fail cleanly
* Fixed failure exits to reflect that their was an issue instead of exiting cleaning
* Refactored and updated documentation
* Automation to actually package the need files on each tag, and fixed hardcoded reference to older package
* Actually released  fluent-plugin-zebrium_output-1.60.1 to rubygems, as 1.60 is a bad build and needs to be yanked. Issue due to lack of automation for releasing.  https://github.com/zebrium/fluentd-output-zebrium/issues/3
* Refactored td-agent-3 to td-agent-4
*
